src/examples/AnimateMasking.cpp

```diff
@@ -0,0 +1,191 @@
+#include "Common.h"
+
+/************************************************************************/
+/* Drawing Commands                                                     */
+/************************************************************************/
+
+tvg::Shape *pMaskShape = nullptr;
+tvg::Shape *pMask = nullptr;
+
+static bool rendered = false;
+static int count = 0;
+static int frame = 0;
+static double t1, t2, t3, t4;
```

Comment:
These values are unused.

Reply:
I removed those unused values, thanks.
Regarding problems in loading SVG, you are right - gradients with masks are not working and the same with loding example like Skeleton and its left eye.